usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/googleanalytics/GoogleAnalyticsEventSender.java

```diff
@@ -55,7 +42,7 @@ public UsageRequest(IJBossToolsEclipseEnvironment environment) {
 	 * @param startNewVisitSession if false, the current session from environment is used
 	 * @return true if the request was sent successfully
 	 */
-	synchronized public boolean sendRequest(String pagePath,
+	public synchronized boolean sendRequest(IJBossToolsEclipseEnvironment environment, String pagePath,
```

Comment:
because the SPI is now an interface so you cannot force this to be used in a constructor